doc/api/net.md

```diff
@@ -902,6 +905,20 @@ For TCP connections, available `options` are:
   **Default:** `false`.
 * `keepAliveInitialDelay` {number} If set to a positive number, it sets the initial delay before
   the first keepalive probe is sent on an idle socket.**Default:** `0`.
+* `autoSelectFamily` {boolean}: If set to `true`, it enables a family autodetection algorithm
+  that loosely implements section 5 of [RFC 8305][].
+  The `all` option passed to lookup is set to `true` and the sockets attempts to connect to all
+  obtained IPv6 and IPv4 addresses, in sequence, until a connection is established.
+  The first returned AAAA address is tried first, then the first returned A address and so on.
```

Comment:
This sentence is a bit vague. If multiple `AAAA` addresses are returned and multiple `A` addresses are returned, is the sequence tried `AAAA`, then `A`, then `AAAA`, ... or does it try all of the `AAAA` addresses first then all of the `A`?